hicommonwealth/commonwealth
# Clean up snapshot

## Link to Issue
Closes: #5252 

## Description of Changes
- Removes `SnapshotSpaces` and `SnapshotProposal` models
- Replaces `CommunitySnapshotSpaces` table with `Communities.snapshot_spaces` column
- Add `snapshot` service in `libs/model` so with some utility functions

## Test Plan
- View the `dydx` community and ensure they still have snapshot enabled
- Go to a community you are an admin of (or set yourself as super-admin) and add the 'dydxgov.eth' snapshot space to it.
- Refresh to confirm it was saved successfully then remove it and refresh again.

## Deployment Plan

1. 

## Other Considerations

-
